testkit/src/main/scala/org/bitcoins/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -338,19 +339,19 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
     val txsF = for {
       blocks <- sender.generate(2)
       firstBlock <- sender.getBlock(blocks(0))
-      firstTransaction <- sender.getTransaction(firstBlock.tx(0))
+      firstTransaction <- sender.getRawTransaction(firstBlock.tx.head)
```

Comment:
We were having issues with `gettransaction` sporadically failing, either by missing the `blockindex` field or getting `Invalid or non-wallet transaction id (code -5)`

Is this problematic? From what I can tell it should be fine